class-components/src/components/BottomSection/BottomSection.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
+import { useSearchParams, useNavigate, Outlet } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useGetPeopleQuery } from '../../store/apiSlice';
+import { RootState, AppDispatch } from '../../store/store';
+import {
+  setSelectedItemId,
+  setSelectedItem,
+  setItems,
+  toggleSelectedItem,
+} from '../../store/searchSlice';
+import Flyout from '../Floyt/Flyout';
+import { People } from './BottomSection.type';
+import Pagination from '../Pagination/Pagination';
+import './BottomSection.css';
+
+const BottomSection: React.FC = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const searchTerm = useSelector((state: RootState) => state.search.searchTerm);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const navigate = useNavigate();
+  const selectedItems = useSelector((state: RootState) => state.search.selectedItems);
+
+  const currentPage = Number(searchParams.get('page')) || 1;
+  const { data, error, isLoading } = useGetPeopleQuery({ search: searchTerm, page: currentPage });
+  const [selectedItem, setSelectedItemLocal] = useState<People | null>(null);
+
```

Comment:
I think it would be better (but it also may be just a matter of taste) if you grouped all these, e.g. by State / Effects / Redux. e.g.

```
const [selectedItem, setSelectedItemLocal] = useState<People | null>(null);
  const [searchParams, setSearchParams] = useSearchParams();
  const navigate = useNavigate();

  const dispatch = useDispatch<AppDispatch>();
  const searchTerm = useSelector((state: RootState) => state.search.searchTerm);
  const selectedItems = useSelector((state: RootState) => state.search.selectedItems);

  const currentPage = Number(searchParams.get('page')) || 1;
  const { data, error, isLoading } = useGetPeopleQuery({ search: searchTerm, page: currentPage });
```
Just a suggestion for the future